src/components/Steps/Steps.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <Steps
+    v-model:activeStep="activeStep"
+    :model="primeVueItems"
+    :pt="{ menu: { class: 'uic-steps-menu' } }"
+    :readonly="false"
+    data-testid="uic-steps"
+  >
+    <template #item="{ active, index, label }">
+      <span :class="['uic-steps-action', { active }]">
+        <span :class="['uic-steps-step', { icon: !!stepIcons[index] }]">
```

Comment:
Nit - entirely personal pref, but I generally prefer explicit casts to double bangs, just for readability.